src/DependencyInjection/Configuration.php

```diff
@@ -51,6 +51,12 @@ public function getConfigTreeBuilder(): TreeBuilder
          */
         $rootNode
             ->addDefaultsIfNotSet()
+            ->children()
+                ->integerNode('cache_ttl')
+                    ->info('The cache TTL')
```

Comment:
```suggestion
                    ->info('The number of seconds before an element is automatically invalidated in the cache')
```

or something better :)